packages/cloudflare/src/cli/templates/cache-handler/cache-store.ts

```diff
@@ -0,0 +1,65 @@
+import type { IncrementalCacheValue } from "next/dist/server/response-cache";
+
+export type CacheEntry = {
+  lastModified: number;
+  value: IncrementalCacheValue | null;
+};
+
+export type CacheStore = {
+  get: (key: string) => Promise<CacheEntry | null>;
+  put: (key: string, entry: CacheEntry, ttl?: number) => Promise<void>;
+};
+
+export function getCacheStore() {
+  const kvName = process.env.__OPENNEXT_KV_BINDING_NAME;
+  if (process.env[kvName]) {
+    return new KVStore(process.env[kvName] as unknown as KVNamespace);
+  }
+
+  return new CacheAPIStore();
+}
+
+const defaultTTL = 31536000; // 1 year
```

Comment:
Can we rename that to `oneYearInMs` to be explicit (we can add a comment saying it's the default cache TTL)